lib/pwnlib/tubes/tube.rb

```diff
@@ -105,9 +108,13 @@ def recvpred(timeout: nil)
               data << c
             end
             data.slice!(0..-1)
-          ensure
+          # rubocop:disable Style/RescueStandardError
```

Comment:
And this must be changed back to ensure

Reply:
I think I finally get what I misunderstood about this code, but I'll raise it as an issue so it can be discussed properly.